content/en/real_user_monitoring/browser/troubleshooting.md

```diff
@@ -102,6 +102,14 @@ If an event or a request goes beyond any of the following limitations, it is rej
 | Maximum event size                       | 256 KB       |
 | Maximum intake payload size              | 5 MB         |
 
+## Customer data is over 3KiB warning
+
+The RUM browser SDK allows you to set [global context](https://docs.datadoghq.com/real_user_monitoring/browser/modifying_data_and_context/?tab=npm#global-context), [user information](https://docs.datadoghq.com/real_user_monitoring/browser/modifying_data_and_context/?tab=npm#user-session) and [feature flags](https://docs.datadoghq.com/real_user_monitoring/guide/setup-feature-flag-data-collection/?tab=browser) which are then included with the collected events.
+
+To prevent impacting the user bandwidth the RUM browser SDK throttles the data sent to the Datadog intake. However, to be able to handle heavy RUM traffic it does not cover lower connectivity (<4g). If a large volume of data is sent on a low-connectivity network, it can slow down the requests made by the host application.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏That is not clear to me, what do you mean by "it does not cover lower connectivity"?

Reply:
Agreed. It might be interpreted as we don't support lower connectivities.

Reply:
Is something like this accurate?
"Note: sending large volumes of context data can impact performance for users on slow internet connections."

However, I'm also wondering if we should directly update the console message instead of adding this to our documentation. What do you think?

Reply:
I've updated the documentation according to the RFC.